<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTimerItemDlgClass</class>
 <widget class="QDialog" name="CreateTimerItemDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建计时器项</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>计时器名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_timer_name">
        <property name="placeholderText">
         <string>按自己喜好或需求填即可</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>要绑定的进程名称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_proc_name">
        <property name="placeholderText">
         <string>按住右侧按钮移动靶心, 可自动获取进程名称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="btn_aim_process">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;自动捕获器&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;长按拖拽&lt;/span&gt;到目标窗体上, 自动识别进程&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/pictures/aim.png</normaloff>:/pictures/aim.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>计时器标签</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edit_tags">
        <property name="placeholderText">
         <string>标签可用于分类计时器</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="chk_can_del">
        <property name="text">
         <string>是否可被删除(一经创建无法修改)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="chk_can_pause">
        <property name="text">
         <string>是否可被暂停(一经创建无法修改)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="chk_can_edit">
        <property name="text">
         <string>是否可被编辑(一经创建无法修改)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="chk_start_imm">
        <property name="text">
         <string>创建后立即运行</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateTimerItemDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
